app/src/main/java/app/revanced/integrations/patches/MicroGSupport.java

```diff
@@ -26,6 +26,7 @@ public static void checkAvailability() {
             Toast.makeText(context, str("microg_not_installed_warning"), Toast.LENGTH_LONG).show();
 
             var intent = new Intent(Intent.ACTION_VIEW);
+            intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Please clarify your PR and this line.

Reply:
In the check, the app would exit with the error `Calling startActivity() from outside of an Activity  context requires the  
FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?` and setting the flag `FLAG_ACTIVITY_NEW_TASK` would make the Vanced MicroG release link open in a browser. 

Source: https://stackoverflow.com/questions/3918517/calling-startactivity-from-outside-of-an-activity-context